internal/contact/email.go

```diff
@@ -0,0 +1,43 @@
+package contact
+
+import (
+	"context"
+	"fmt"
+	"github.com/CollActionteam/collaction_backend/internal/constants"
+	"github.com/CollActionteam/collaction_backend/internal/models"
+)
+
+const separator = "### app version:"
+
+type EmailRepository interface {
+	Send(ctx context.Context, data models.EmailData) error
+}
+
+type ConfigManager interface {
+	GetParameter(name string) (string, error)
+}
+
+type Contact struct {
```

Comment:
This type should be private

Reply:
whey? can you explain your reason?
based on the https://github.com/CollActionteam/collaction_backend/pull/62#discussion_r782858417 if we not returning the interface the struct must be public to be able to use it and test it (testing the service as an external package, not a private package, it is a best practice to write unit tests as a real user to that package and because of that when we are writing our test we can have a separate package name like contact_test, to be more realistic, this is the only case we can game multi-package name inside a single package/directory)

Reply:
I think you need to return the ContactService interface and put this struct as private. I put a comment #62 comment.
See if you approve of this approach.